drop/action.yml

```diff
@@ -5,9 +5,10 @@ name: 'Drop binaries to runner'
 description: 'Download binaries securely into the runner'
 inputs:
   insecure:
-    type: boolean
     description: Don't attempt to find and apply security policies
-  binary:  # id of input
+    required: false
+    default: 'false'
```

Comment:
if this is going to be optional, then should it default to true?

Reply:
that is the insecure, maybe we want to get rid of that when is stable enough